DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Queries/Implementations/GitHubQuery.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtension.Statistics.WebApi.CommunicationModels;
+using GitHubExtension.Statistics.WebApi.Mappers.HttpRequest;
+using GitHubExtension.Statistics.WebApi.Queries.Constant;
+using GitHubExtension.Statistics.WebApi.Queries.Interfaces;
+using Newtonsoft.Json;
+
+namespace GitHubExtension.Statistics.WebApi.Queries.Implementations
+{
+    public class GitHubQuery : IGitHubQuery
+    {
+        #region fields
+        private readonly HttpClient _httpClient;
+        #endregion
+
+        public GitHubQuery()
+        {
+            this._httpClient = new HttpClient();
+        }
+
+        #region methods
+        public async Task<List<int>> GetUserCommitsInYear(string owner, string repository, string token)
```

Comment:
Nice method splitting. May be you can use all this method directly from the client and skipping any transformations on server side?